Apps/Admin/Client/Properties/launchSettings.json

```diff
@@ -1,14 +1,13 @@
 {
```

Comment:
Our .json files should have 4 spaces for indentation according to our EditorConfig settings - my Rider picks this up automatically.  (Applies to other `launchSettings.json` changes as well.)